food/examples/2.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": 2,
+  "text": "Cheese, glorious cheese!\nBy: Sheri Wetherell\nPublished: December 10, 2007\n\nI just visited Artisanalcheese.com and read about the various New York City wine and cheese events that are happening this December and January. Everything from Cheese and Wine 101 to Sake and Artisinal Cheese Pairings.  Who'da thunk sake and cheese!? Love it!  This inspired me. With family coming to town for the holidays I think it will be fun to do wine and cheese pairings in lieu of the ordinary hors d'oeuvres spread. Trader Joe's always has an interesting selection of cheeses (most often some of the same cheeses that are sold at a premium at higher-end grocers) that don't break the bank. If you have a TJ's in your area I recommend starting there then perhaps fill in your selection with some harder-to-find varieties to keep it interesting.\nFor my pairings I'll offer a selection of European and domestic cheeses to keep it eclectic. I recognize that all palates are different and some may disagree with my selections but here are few of my pairing thoughts...\nManchego, a classic semi-firm Spanish sheep's milk cheese, will go well with a Tempranillo.\n Stilton, not everyone's favorite, but a good stinky and bitey blue I think is a must for any cheese platter.  Stilton is best paired, in my humble opinion, with something robust such as Sirah, or sweet like a Tawny Port or Gewurztraminer.\nBrillat Savarin, a thick and velvety French cow's milk cheese, will be nice with a Chardonnay or, if keeping to reds, a Cabernet-Franc.\nAn extra-sharp domestic Cheddar, such as Vermont's Cabot, will pair well with a hearty Rioja or Cabernet.\nTo cleanse the palate I'll add some crisp red grapes or halved figs and almonds drizzled with honey.\nI'd love to hear some of your thoughts as well!\nCheers and good eats!\n\nComments:\nKen Wetherell\nDecember 15, 2007\n\nMmmmm...Cabot!  Got'ny cheeeeeez?  Nice post.  Thanks.",
+  "source": "foodista",
+  "added": "2024-05-07T19:57:00.094171",
+  "created": "2007-12-10T00:00:00",
+  "metadata": {
+    "license": "Creative Commons - Attribution - https://creativecommons.org/licenses/by/3.0/",
+    "url": "https://foodista.com/blog/2007/12/10/cheese-glorious-cheese",
+    "authors": [
+      [
+        "Ken Wetherell",
+        ""
```

Comment:
Should the author list have more structure? For the purposes of attribution I think we only need the author name as displayed, fwiw, and not the user id or anything.

Reply:
I updated it to only output the author names for now. The parser still finds the user ids (and is robust if it fails to) so we can turn it on in the future.

For Stack Exchange our author list includes both the user names and the user link but they are a single list, not grouped like this one was. I don't think it is worth making sure they all match exactly. IIRC, someone said they we technically don't even need specific authors for attribution, just the site it comes from :woman_shrugging:

Reply:
Yeah, I'm not sure there is a specific set of practices, but CC has some recommendations that do include crediting the author: https://wiki.creativecommons.org/wiki/Recommended_practices_for_attribution 

The reason the SE parsing includes author profile page links is that SE actually requests even more stuff for attribution: https://stackoverflow.blog/2009/06/25/attribution-required/ so it seemed like a good practice to try to include that. Unless foodista makes a similar request, I think we can avoid including the user page and the name is enough.